src/os/bluestore/BlueStore.cc

```diff
@@ -5598,14 +5598,14 @@ int BlueStore::_do_write(
     if (p->second.has_flag(bluestore_extent_t::FLAG_UNWRITTEN)) {
       derr << __func__ << " left behind an unwritten extent, out of sync with "
 	   << "_do_allocate" << dendl;
-      _dump_onode(o);
+      _dump_onode(o, 0);
```

Comment:
Any reason why to invoke this as _dump_onode(o, 0) rather than _dump_onode(o, 30) which would preserve the earlier behaviour?